queue/abstract.lua

```diff
@@ -393,17 +413,47 @@ local function make_self(driver, space, tube_name, tube_type, tube_id, opts)
     return self
 end
 
+--- Release all session tasks if the session has no actual connections.
+local function cleanup_session_tasks(queue_uuid)
+    local sesion_ids = box.space._queue_session_ids.index.queue_uuid:select(
+        queue_uuid,
+        {limit = 1})[1]
+    if sesion_ids ~= nil then
+        return
+    end
```

Comment:
It is a bit tricky, I would leave a comment here. I mean, I looked to the code a few minutes until I got the idea.

Reply:
Done.